types/jbuild

```diff
@@ -0,0 +1,21 @@
+(jbuild_version 1)
+
+(library
+  ((name        mirage_types)
+   (public_name mirage-types)
+   (modules     (mirage_types))
+   (libraries   (mirage-device mirage-time mirage-random mirage-flow
+                 mirage-console mirage-block mirage-clock mirage-net
+                 mirage-kv mirage-fs mirage-channel
+                 mirage-protocols mirage-stack))))
+
+(library
+  ((name        mirage_types_lwt)
+   (public_name mirage-types-lwt)
```

Comment:
In practice that doesn't matter as there is only one module with that name, but it's probably saner to add `(wrapped false)`. Thanks for noticing :-)